src/main/java/com/rtb/projectmanagementtool/task/TaskServlet.java

```diff
@@ -31,6 +34,20 @@ public TaskServlet(DatastoreService datastore) {
   public void doGet(HttpServletRequest request, HttpServletResponse response)
       throws ServletException, IOException {
 
+    // Authenticate
+    AuthOps auth = new AuthOps(datastore);
+    auth.loginUser(request, response);
```

Comment:
Don't need to login here.  Login servlet should handle login.  all other should just do whichUser then redirect if there's no user logged in